lang/python/python-dpkt/Makefile

```diff
@@ -0,0 +1,49 @@
+#
+# Copyright (C) 2017 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=python-dpkt
+PKG_VERSION:=1.91
+PKG_RELEASE:=1
+PKG_MAINTAINER:=Andrew McConachie <andrew@depht.com>
+PKG_LICENSE:=BSD-3-Clause
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/kbandla/dpkt.git
```

Comment:
I've updated the copyright notice to assign copyright to Andrew McConachie.

The reason I draw from github.com/kbandla/dpkt instead of the python repo is because it's more upto date. Specifically, I made some changes to the SSL/TLS implementation, which I need on OpenWRT, and those changes are not in the Python repo version.

Having said that however, I will maintain this and update as necessary in the future as the code progresses. But right now I think it makes sense to draw from the Github repo.